Corona-Warn-App/src/test/java/de/rki/coronawarnapp/nearby/modules/diagnosiskeyprovider/DefaultDiagnosisKeyProviderTest.kt

```diff
@@ -62,135 +51,64 @@ class DefaultDiagnosisKeyProviderTest : BaseTest() {
     )
 
     @Test
-    fun `legacy key provision is used on older ENF versions`() {
-        coEvery { googleAPIVersion.isAtLeast(GoogleAPIVersion.V16) } returns false
+    fun `key provision is used on older ENF versions`() {
```

Comment:
Missleading function name, we are checking that we DO NOT submit keys if the ENF version is too low.

Reply:
changed to fun _provide diagnosis keys with outdated ENF versions_